### PR TITLE
Initial attempt to map barcode scanner

### PR DESCRIPTION
**Pull request by forki**
See https://github.com/ideacreation/react-native-barcodescanner

Does it make sense?

### CODE REVIEW

**Comment by alfonsogarciacaro** [comment]
Looks good to me :+1:

File: react-native-barcodescanner/Fable.Import.ReactNativeBarcodeScanner.fs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    | Front
+    | Back
+
+
+    and BarcodeScanner =
+        BarcodeScannerStatic
+
+
+    type Globals =
+        [<Import("BarcodeScanner", "react-native-barcodescanner")>] static member BarcodeScanner with get(): BarcodeScannerStatic = failwith "JS only" and set(v: BarcodeScannerStatic): unit = failwith "JS only"
```
Comment (alfonsogarciacaro) [inline react-native-barcodescanner/Fable.Import.ReactNativeBarcodeScanner.fs]: The error may be here. According the [BarcodeScanner usage instructions](https://github.com/ideacreation/react-native-barcodescanner#usage) you must import the **default** member of "react-native-barcodescanner". This is a bit tricky in ES6 imports, check the difference between these two import statements:

```js
// Import **member** BarcodeScanner from 'react-native-barcodescanner'
import { BarcodeScanner } from 'react-native-barcodescanner';
// Fable [<Import("BarcodeScanner", "react-native-barcodescanner")>]

// Import **default** member from 'react-native-barcodescanner'
// (and give it the name 'BarcodeScanner' in current file)
import BarcodeScanner from 'react-native-barcodescanner';
// Fable: [<Import("default", "react-native-barcodescanner")>]
```

So here you probably need to change the attribute to `[<Import("default", "react-native-barcodescanner")>]`.

Comment (forki) [inline react-native-barcodescanner/Fable.Import.ReactNativeBarcodeScanner.fs]: It's generating 

    import react_native_barcodescanner from "react-native-barcodescanner";

and that doesn't work

Comment (forki) [inline react-native-barcodescanner/Fable.Import.ReactNativeBarcodeScanner.fs]: scratch that. that works (but is a bit ugly). I had to set flex to make it visible.

**Pull request merged**